clients/spark/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.14.1 - 2024-07-04
+* Bug fix when using repository listing params (#7955)
```

Comment:
I would prefer to rephrase, as a user I am not sure what this means.

Perhaps something like this
```suggestion
* When scanning across all commits, correctly handle lakeFS storage namespaces that do not end in `/` (#7955)
```
?